include/dlaf/blas/tile.h

```diff
@@ -485,7 +502,7 @@ void trsm(cublasHandle_t handle, const blas::Side side, const blas::Uplo uplo, c
 #ifdef DLAF_WITH_CUDA
       blasToCublasCast(a.ptr());
 #elif defined(DLAF_WITH_HIP)
-      // The hipblas API requires a non-const argument
+      // The rocblas API requires a non-const argument
```

Comment:
Did you check by the way if the restriction is really there for rocblas as well, or if it was only a problem with the hipblas API?

Reply:
And if the restriction is still there: is it guaranteed that values of A are not changed?